package.json

```diff
@@ -67,15 +76,13 @@
   "scripts": {
     "lint": "eslint src/ test/ scripts/ examples/",
     "clean": "rimraf src/en.js src/locale-data/ lib/ dist/",
-    "build:data": "babel-node scripts/build-locale-data.js",
+    "build:data": "babel-node scripts/build-locale-data",
     "build:lib": "babel src/ --ignore src/locale-data/ --out-dir lib/",
-    "build:dist:main": "browserify src/react-intl.js -s ReactIntl -g [envify --NODE_ENV development] -t babelify -t browserify-shim -d | exorcist -b ./ -r react-intl:/// dist/react-intl.js.map > dist/react-intl.js",
-    "build:dist:main:min": "browserify src/react-intl.js -s ReactIntl -g [envify --NODE_ENV production] -t babelify -t browserify-shim | uglifyjs -c -m > dist/react-intl.min.js",
-    "build:dist:with-data": "browserify src/react-intl-with-locales.js -s ReactIntl -g [envify --NODE_ENV development] -t babelify -t browserify-shim -d | exorcist -b ./ -r react-intl:/// dist/react-intl-with-locales.js.map > dist/react-intl-with-locales.js",
-    "build:dist:with-data:min": "browserify src/react-intl-with-locales.js -s ReactIntl -g [envify --NODE_ENV production] -t babelify -t browserify-shim | uglifyjs -c -m > dist/react-intl-with-locales.min.js",
-    "build:dist": "npm run build:dist:main && npm run build:dist:main:min && npm run build:dist:with-data && npm run build:dist:with-data:min",
+    "build:dist:dev": "NODE_ENV=development babel-node scripts/build-dist",
+    "build:dist:prod": "NODE_ENV=production babel-node scripts/build-dist",
+    "build:dist": "npm run build:dist:dev && npm run build:dist:prod",
```

Comment:
The development build contains many warns which are removed in the production build.